src/frontend/src/handler/create_source.rs

```diff
@@ -724,6 +724,12 @@ pub async fn handle_create_source(
         )));
     }
 
+    if stmt.source_schema == SourceSchema::Json && stmt.columns.is_empty() {
+        return Err(RwError::from(InvalidInputSyntax(
+            "Json schema definition not specified".to_owned(),
```

Comment:
I'm not sure whether users can understand the error. 

What about sth like
```suggestion
            "column definitions are required for ROW SCHEMA JSON".to_owned(),
```
Better ideas are welcomed.

Reply:
I used the wording `schema definition` because it is mentioned in [documentation](https://www.risingwave.dev/docs/current/create-source-kafka/), but I do agree that `column definition` is more understandable to users. Maybe we can mention both, like `schema (column) definition is required for ROW FORMAT JSON`?